### DISCUSSION TITLE
Pangeo showcase talk on xcdat?

### DISCUSSION THREAD

**Discussion by rsignell-usgs** [xCDAT/xcdat#399] [4. General]
Would someone from this community be interested in giving a 15 minute [Pangeo Showcase](https://pangeo.io/pangeo-showcase.html) talk on xcdat?    

Current functionality and plans would be great, or really anything you think the Pangeo community would be interested in!   

(We took a break on Showcase talks for a few months but we will be starting up again in a few weeks and I'm getting speakers lined up)

**Comment by tomvothecoder** [comment]
Hi Rich, thanks for reaching out about this opportunity. I would happy to give a Pangeo Showcase talk on `xcdat`!
Including the topics you mentioned, I'll discuss with the core team to see what might be cool to share.

Please let me know which slots are available and I'll confirm which works best.

**Reply by tomvothecoder** [reply]
I submitted xCDAT for the a [Pangeo Showcase](https://pangeo.io/pangeo-showcase.html) slot between March 15 through June!
